src/controllers/videoosd.js

```diff
@@ -1070,12 +1075,21 @@ define(["playbackManager", "dom", "inputManager", "datetime", "itemHelper", "med
             }
         }
 
+        /**
+         * Keys used for keyboard navigation.
+         */
+        var NavigationKeys = ["ArrowLeft", "ArrowRight", "ArrowUp", "ArrowDown"];
+
         function onWindowKeyDown(e) {
             if (!currentVisibleMenu && 32 === e.keyCode) {
                 playbackManager.playPause(currentPlayer);
                 return void showOsd();
             }
 
+            if (layoutManager.tv && NavigationKeys.indexOf(e.key) != -1) {
+                return void showOsd();
+            }
```

Comment:
We should just call `showOsd()` in these two blocks and then return normally, unless this provides some benefit. I think it's just left over from the minified code.